### DISCUSSION TITLE
Cmit /tfly all

### DISCUSSION THREAD

**Discussion by TimHarrio** [Zrips/CMI#3158] [Feature Requests]
Add the possibility of giving the tfly for everyone.

This can be useful for VoteParty, for example giving the flyer to all players.

Exemple :
/cmi tfly all +10s

**Comment by CloudeLecaw** [comment]
wanne create an command alias?
Command aliases support !allplayers

>  /cmi aliaseditor new [alias]-[original command]

**/cmi aliaseditor new flyall-!allplayers cmi tfly $1- -s**

this would allow you to give all players tfly with an amount of time
example: flyall 10        <- all get tfly for 10 secounds

you also can setup a fixed amount of time:
**/cmi aliaseditor new flyall90-!allplayers cmi tfly 90 -s**
when executing tflyall90 everyone gets now 90 secounds tfly

**Comment by Zrips** [comment]
Correction
**/cmi aliaseditor new flyall90-allPlayers! cmi tfly [allPlayers] 90 -s**

**Comment by TimHarrio** [comment]
thank you =)
